mtbf_driver/mtbf.py

```diff
@@ -16,10 +16,33 @@
 from utils.step_gen import RandomStepGen, ReplayStepGen
 from utils.time_utils import time2sec
 
+import mozversion
+
+
+class MTBFTestRunner(GaiaTestRunner):
+
+    saved_version_info = None
+
+    def get_version_info(self, input_version_info=None):
+        if input_version_info is None:
+            self.saved_version_info = mozversion.get_version(binary=self.bin,
+                                                             sources=self.sources,
+                                                             dm_type=os.environ.get('DM_TRANS', 'adb'),
+                                                             device_serial=self.device_serial)
+            mozversion.get_version = self._new_get_version_info
+        else:
+            self.saved_version_info = input_version_info
+            mozversion.get_version = self._new_get_version_info
```

Comment:
I think yes....in case the case runner invoke the get_version before or after run_test